packages/react/src/progress-indicator/ProgressIndicator.tsx

```diff
@@ -15,14 +12,14 @@ import {
 } from './utils';
 
 export type ProgressIndicatorProps = {
-	/** The list of items, including level 2 items, to display. Use in conjunction with `activePath`. Note level 2 items are only available when using links. */
-	items: (ProgressIndicatorItem | ProgressIndicatorItemWithLevelTwoItems)[];
 	/** Used for highlighting the active step. This should match the active item's `href` (or label if no href is supplied). */
 	activePath?: string;
 	/** If the ProgressIndicator is placed on a page with 'bodyAlt' background, please set this to "bodyAlt". */
 	background?: 'body' | 'bodyAlt';
 	/** If true, the `subTitle` above the list of items will not be rendered. */
 	hideSubtitle?: boolean;
+	/** The list of items, including level 2 items, to display. Use in conjunction with `activePath`. Note level 2 items are only available when using links. */
+	items: (ProgressIndicatorItem | ProgressIndicatorItemWithLevelTwoItems)[];
```

Comment:
The reason this prop was at the top was to give more significance to required props. For me that's preferable but it's not a big deal if we want to have a single alpha order. We should just pick one way, though.

Reply:
I think just alphabetical since it's the easiest and is already automated in our editor and in our docs

Reply:
Ok, cool cool, I'll do this from now on.